content/pages/edit.json

```diff
@@ -20,6 +20,15 @@
         "rows": 10,
         "required": true
       },
+      {
+        "name": "price",
+        "label": "New Price",
+        "type": "number",
+        "min": "1",
+        "placeholder": "e.g. Shapes of Desert Plants",
```

Comment:
we can just use `"0"` as placeholder for the price input

Reply:
right, missed that since we won't have a case when that would pe displayed